src/base/TilesRendererBase.js

```diff
@@ -557,10 +559,10 @@ export class TilesRendererBase {
 
 						}
 
-						const uri = parseTile.content.uri;
+						const contentType = getTileContentType( buffer );
 						const extension = getUrlExtension( uri );
```

Comment:
this is actually a change in behaviour: this now uses the result of preprocessURL to get the extension instead of the original url. Which one is more correct?

Reply:
I think using the result of `preprocessURL` is good - it's really just there to let people add url parameters, etc to the result.